lib/puppet_x/puppetlabs/powershell/powershell_manager.rb

```diff
@@ -9,14 +9,14 @@ module PowerShell
     class PowerShellManager
       @@instances = {}
 
-      def self.instance(cmd, debug = false)
+      def self.instance(cmd, debug = false, pipe_timeout = 30)
```

Comment:
Do we want defaults on the factory here and on the constructor?

Reply:
I think the default is only necessary here, not on the constructor, unless I'm missing something / there's a way to call the instructor so it won't inherit from here.

Reply:
Fixed in the latest push.